conservation_hmm/README.md

```diff
@@ -0,0 +1,35 @@
+# conservation\_hmm
+
+## About
+
+`conservation_hmm.py` is a Python script which can be used to calculate per-position information content (IC) values for amino acid residues in a FASTA file. It is inspired by the [INTAA-conservation](https://github.com/davidjakubec/INTAA-conservation) pipeline utilized by the [Amino Acid Interactions (INTAA) web server](https://bioinfo.uochb.cas.cz/INTAA/). Unlike INTAA-conservation, `conservation_hmm.py` skips parsing the PDB structure and works with the single-sequence FASTA files directly. For further information on the pipeline's logic, please refer to the [INTAA manual](https://ip-78-128-251-188.flt.cloud.muni.cz/energy/doc/manual2.html#Calculation_of_information_content).
+
+## Installation
+
+`conservation_hmm.py` is a transferable, executable script which does not require any installation. It does not depend on any other Python packages or files. For other dependencies, please see the section below.
+
+### Requirements
+
+The [HMMER](http://hmmer.org/) software package, including the Easel tools, must be installed and the programs must be available in `$PATH`. All development and testing is done using HMMER 3.3.2.
+
+The script `examples/database/download_Swiss-Prot.sh` can be used to download an example sequence database.
+
+## Usage
+
+Use
+```
+conservation_hmm.py FASTA_file database_file working_directory target_file
```

Comment:
All the arguments are positional, can't we use arguments like ```--fasta-file```, ```--working-directory```.
Also I would move this into the python script and here show example of commands to run, including the download command.

As user can then type ```conservation_hmm.py -h``` to get the help. That seems better to me, as I can get the help when and where I need it, in the terminal.